packages/portal/src/i18n/lang/en.js

```diff
@@ -1178,6 +1180,10 @@ export default {
     },
     "notifications": {
       "deleted": "Your gallery has been deleted.",
+      "itemAdded": "Item has been added to gallery {gallery}.",
```

Comment:
Let's make this:
`"itemAdded": "Item has been added to the gallery \"{gallery}\".",`

Reply:
Done.